bin/node-template/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.2"
```

Comment:
I don't have anything against it but why a docker compose for a single service? 

Now that we have a compose file, we could bundle this with a UI image as well.

Reply:
Indeed. Let me deal with this separately in this issue:
https://github.com/substrate-developer-hub/substrate-node-template/issues/163